src/pybind/mgr/balancer/module.py

```diff
@@ -1275,6 +1287,16 @@ def execute(self, plan):
             }), 'foo')
             commands.append(result)
 
+        for pgid in incdump.get('old_pg_upmap_items', []):
```

Comment:
@liewegas  I take the opportunity to change the order we issue add/rm commands. Because this matches the order we apply upmaps into osdmap better (see OSDMap::apply_incremental). In the worse case if balancer did something bad, e.g., try to add and then rm upmap result for the same pg, this way we can still handle it nicely.